test/timezone/lookup/test_geonames.rb

```diff
@@ -9,10 +9,19 @@ def coordinates
     [-34.92771808058, 138.477041423321]
   end
 
-  def lookup
+  def etc_data
+    {
+      arctic: { coordinates: [89, 40], name: 'Etc/GMT+3' },
+      atlantic: { coordinates: [0, 0], name: 'Etc/UTC' },
+      norfolk: { coordinates: [-29, 167], name: 'Etc/GMT+11' }
+    }
+  end
+
+  def lookup(options = {})
```

Comment:
Maybe we can just yield here to make additional option setting easier:

```
def lookup(&block)
  ...
  config.request_handler = HTTPTestClient
  yield(config) if block_given?
  
  Timezone::Lookup::Geonames.new(config)
end
```

then

```
mine = lookup { |c| c.offset_etc_zones = trues }
```